communication/tests/catch/protocol.cpp

```diff
@@ -66,11 +66,11 @@ void event_handler(const char* event, const char* data)
 {
 }
 
-SCENARIO("5 subscribe messages are registered")
+SCENARIO("6 subscribe messages are registered")
```

Comment:
Shouldn't you use MAX_SUBSCRIPTIONS here and in the test as well?

Reply:
Yes probably :)  We'll have to fix it going forward.

Reply:
Might be hard to include protocol_defs.h though... not sure.  At least if the number changes the test will fail.  That said, this test was/is broken and fixed in PR https://github.com/particle-iot/device-os/pull/1524 so we need to get that fixed up and merged asap.